node/channel.js

```diff
@@ -319,6 +319,11 @@ TChannel.prototype.listen = function listen(port, host, callback) {
 TChannel.prototype.register = function register(name, options, handler) {
     var self = this;
 
+    // If we are the top channel
+    if (!self.topChannel) {
```

Comment:
`topChannel` is a channel instance that is a reference to the top channel;

all sub channels have a `topChannel` field that is the top channel; the top channel does not have a top channel; i..e `topChannel` is null.

I can do explicit null checks to avoid boolean confusion